extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -65,16 +67,16 @@ class AdSenseNetworkConfig {
 
   /** @override */
   getConfigUrl() {
+    const docInfo = documentInfoForDoc(this.autoAmpAdsElement_);
+    const canonicalHostname = parseUrl(docInfo.canonicalUrl).hostname;
```

Comment:
We only care about the hostname so multiple AMP docs pointing to the same URL shouldn't matter. The canonical hostname is what things will be keyed under server-side, since that's what is used in the ad requests, so the config request has to match this.